### ISSUE TITLE
prefer-default-export false negative with named exports

### ISSUE THREAD

**Issue by FredyC** [benmosher/eslint-plugin-import#609]
I have a simple file `Memory.js` which contains:

```js
import t from 'tcomb'
import MemoryKey from './MemoryKey'

const Memory = t.declare('Memory')

export const MemoryValue = t.union([ Memory, t.String, t.Boolean, t.Number,], 'MemoryValue')
export default Memory.define(t.dict(MemoryKey, MemoryValue))
```

Then in `index.js` which exports all the types, I have it like this...

```js
export Memory, { MemoryValue } from './Memory' // eslint-disable-line import/prefer-default-export
export MemoryDictionary from './MemoryDictionary'
export MemoryKey from './MemoryKey'
```

The rule `prefer-default-export` is failing on that line for some reason. Babel transpiles that code just fine.

> eslint: 3.7.0
> eslint-plugin-import: 1.16.0
> eslint-config-airbnb-base: 8.0.0

**Comment by benmosher** [comment]
Are you using babel-eslint?

**Comment by FredyC** [comment]
Yes, of course. Not sure why I omitted that one from the list :)

> babel-eslint: 7.0.0

**Comment by benmosher** [comment]
Sounds like a bug, then. Will take a look 😄

**Comment by jfmengels** [comment]
Fixed in #622 :)

**Comment by FredyC** [comment]
Awesome, thanks ! :)
